95a50c7

```diff
@@ -0,0 +1,45 @@
+[33mfbdb95c[m[33m ([m[1;36mHEAD -> [m[1;32mmain[m[33m, [m[1;31morigin/main[m[33m, [m[1;31morigin/HEAD[m[33m)[m HEAD@{0}: reset: moving to HEAD
```

Comment:
What is this?

Reply:
I mean why git reflog is committed?

Reply:
I need to add something to my .gitignore to prevent this?

Reply:
I don't know how you get that into commit, it looks like regular file. Have you accidentaly saved git reflog into file?

Reply:
`.git/` folders are ignored by default